frontend/src/styles/training-page.scss

```diff
@@ -1,79 +1,67 @@
-.register {
-	margin-top: 5vh;
+.trainingPageContainer {
+	margin-top: 2vh;
 
-	.registerContainer {
+	.alertContainer {
 		display: flex;
-		align-items: center;
+		justify-content: center;
+		margin-bottom: 4rem;
 
-		.imgContainer {
-			img {
-				display: none;
-
-				@media screen and (min-width: 768px) {
-					display: block;
-				}
-			}
-		}
-	}
-
-	.registerContent {
-		width: 400px;
-		min-height: 550px;
-		display: flex;
-		flex-direction: column;
-		border-radius: 19px;
-		margin: 5vh auto;
-		// border: 2px solid #f6921e;
-		background: #faf3ed;
-		padding: 2rem;
-		box-shadow: 10px 10px 8px $secondary;
-
-		.formH1 {
+		.alert {
+			z-index: 1;
+			width: 60%;
 			text-align: center;
-			margin-bottom: 1rem;
-		}
-
-		.registerForm {
-			display: flex;
-			flex-direction: column;
+			align-self: center;
+			@media screen and (max-width: 1500px) {
+				width: 90%;
+			}
 
-			label {
-				font-size: 1rem;
-				font-weight: 300;
+			.fa-solid {
+				margin: 0 1rem;
 			}
 
-			select {
-				padding: 0.5rem 0;
-				margin: 1rem 0;
-				border-radius: 5px;
+			span {
+				margin: 0 0.3rem;
 			}
+		}
+	}
 
-			input {
-				margin-bottom: 0.5rem;
-				outline: none;
-				border: none;
-				padding: 0.5rem;
-				background: #f1e9e9;
-				border-radius: 12px;
-				font-weight: 400;
+	h2 {
+		margin: 2rem 4rem;
+	}
+	.onGoingTraining {
+		margin-left: 4rem;
+		margin-bottom: 4rem;
 
-				&:focus {
-					background: #f4b876;
-					transition: 1s ease-in;
-				}
+		.cohort3 {
+			height: 500px;
+			width: 400px;
+			img {
```

Comment:
This is repetition. You could state it globally for images to take a 100% width and height. 

If that's done, the image would automatically span through the parent container without the repetition.